src/spi_comms.c

```diff
@@ -97,7 +97,7 @@ void manage_cmd (uint8_t spi_first_byte, uint8_t spi_second_byte){
 
 
 // calibrate and take well readings
-// well_data[7] - optical density = 0, fluorescent LED = 1
+// well_data[5] - optical density = 0, fluorescent LED = 1
 // well_data[4:0] - well number (0-31)

```

Comment:
Pick a convention for the bit 5 value and stick with it - optical density is for PAY-LED, fluorescence is for PAY-OPT. I recommend putting LED as 0 so we collect that data first since we are expected to use all 32 wells of data.